app/src/main/java/uk/org/openseizuredetector/OsdUtil.java

```diff
@@ -37,6 +37,7 @@ Pebble smart watch (http://getpebble.com).
 import android.database.SQLException;
 import android.database.sqlite.SQLiteDatabase;
 import android.database.sqlite.SQLiteOpenHelper;
+import android.hardware.SensorManager;
```

Comment:
I am not keen on having Android hardware specific imports required by OsdUtil - should we not have the device specific code in the relevant dataSource code and not in a generic utilities function?  (Admittedly, I think there is some Pebble specific code in there at the moment, but I think we should get rid of that too!